attributes/default.rb

```diff
@@ -30,14 +30,23 @@
 default[:tapalcatl][:expvars][:serve] = false
 default[:tapalcatl][:expvars][:log_interval] = 0
 
-# aws specific configuration
-default[:tapalcatl][:aws][:region] = 'us-east-1'
-default[:tapalcatl][:s3][:keypattern] = '/{prefix}/{hash}/{layer}/{z}/{x}/{y}.{fmt}'
-default[:tapalcatl][:s3][:healthcheck] = '/healthcheck.txt'
-
-# these are expected to be set downstream
-default[:tapalcatl][:s3][:bucket] = 'put-your-bucket-name-here'
-default[:tapalcatl][:handler][:s3][:prefix] = 'put-your-prefix-here'
+handler_cfg_json = {
+  'mime' => node[:tapalcatl][:mime],
+  'storage' => {
+    'tmp' => {
+      'type' => 'file',
+      'basedir' => '/tmp',
+      'metatilesize' => 1,
+      'layer' => 'all',
+    },
+  },
+  'pattern' => {
+    '/{z:[0-9]+}/{x:[0-9]+}/{y:[0-9]+}.{fmt}' => {
+      'storage' => 'tmp',
+    },
+  },
+}.to_json()
+default[:tapalcatl][:handler_cfg_json] = handler_cfg_json
```

Comment:
I missed the `.to_json` (parens not necessary) on the line above. Would it be easier to read if the `.to_json` was on the following line instead?